app/jitsi/database.py

```diff
@@ -0,0 +1,21 @@
+from . import db
```

Comment:
I still think this file is unnecessary and these methods can be `User` model class methods

Reply:
or Room

Reply:
This is the global sqlalchemy `db` object. I moved those other methods into the User class